custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -187,6 +187,26 @@ def _get_mapping(field: Fields, values: List[Values]) -> Map:
     return Map({"field": Enum(str(field)), "condition": conditions, "value": value_enum})
 
 
+def check_is_mm(data):
```

Comment:
why do we need this? MM data should be loaded in as a sparse DF.

Reply:
this is how it was read from the transform in some of the functional tests.

Reply:
lets please assert that we're passing around a dataframe with sparse columns, and not a csr matrix, if possible.